docs/openshift/deploy.md

```diff
@@ -0,0 +1,91 @@
+
+## Gather Information
+
+* The subscription in which you would like to provision the cluster. This is a uuid which can be found with `az account list -o table`.
+* Proper access rights within the subscription. Especially the right to create and assign service principals to applications (see ACS Engine the Long Way, Step #2)
+* A `dnsPrefix` which forms part of the the hostname for your cluster (e.g. staging, prodwest, blueberry). The DNS prefix must be unique so pick a random name.
+* A location to provision the cluster e.g. `eastus`.
+
+```
+$ az account list -o table
+Name                                             CloudName    SubscriptionId                        State    IsDefault
+-----------------------------------------------  -----------  ------------------------------------  -------  -----------
+Red Hat OnDemand                                 AzureCloud   225e02bc-43d0-43d1-a01a-17e584a4ef69  Enabled  True
+```
+
+## Deploy
+
+For this example, the subscription id is `225e02bc-43d0-43d1-a01a-17e584a4ef69`, the DNS prefix is `openshift-red`, and location is `eastus`.
+
+Run `acs-engine deploy` with the appropriate arguments:
+
+```
+$ acs-engine deploy --subscription-id 225e02bc-43d0-43d1-a01a-17e584a4ef69 \
+    --dns-prefix openshift-red --location eastus \
+    --api-model examples/openshift.json
+
+INFO[0034] Starting ARM Deployment (openshift-red-1843927849). This will take some time...
+INFO[0393] Finished ARM Deployment (openshift-red-1843927849).
+```
+
+`acs-engine` will output Azure Resource Manager (ARM) templates, SSH keys and a node configuration in `_output/openshift-red` directory:
+
+   * `_output/openshift-red/apimodel.json`
+   * `_output/openshift-red/azuredeploy.json`
+   * `_output/openshift-red/azuredeploy.parameters.json`
+   * `_output/openshift-red/master.tar.gz`
+   * `_output/openshift-red/node.tar.gz`
+
+Administrative note: By default, the directory where acs-engine stores cluster configuration (`_output/openshift-red` above) won't be overwritten as a result of subsequent attempts to deploy a cluster using the same `--dns-prefix`) To re-use the same resource group name repeatedly, include the `--force-overwrite` command line option with your `acs-engine deploy` command. On a related note, include an `--auto-suffix` option to append a randomly generated suffix to the dns-prefix to form the resource group name, for example if your workflow requires a common prefix across multiple cluster deployments. Using the `--auto-suffix` pattern appends a compressed timestamp to ensure a unique cluster name (and thus ensure that each deployment's configuration artifacts will be stored locally under a discrete `_output/<resource-group-name>/` directory).
```

Comment:
Maybe split to separate lines because github reviewing does not help

Reply:
@pschiffe please word wrap everything ;)